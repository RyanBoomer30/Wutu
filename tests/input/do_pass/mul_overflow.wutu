1152921504606846976 * 3