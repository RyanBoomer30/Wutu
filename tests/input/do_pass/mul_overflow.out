3458764513820540928